4-WebApp-your-API/4-3-AnyOrg/Readme.md

```diff
@@ -90,218 +71,289 @@ The Web API:
+    1. Select **optional claim type**, then choose **ID**.
+    1. Select the optional claim **acct**.
+    > Provides user's account status in tenant. If the user is a **member** of the tenant, the value is *0*. If they're a **guest**, the value is *1*.
+    1. Select **Add** to save your changes.
+
+##### Configure the client app (WebApp_MultiTenant_v2) to use your app registration
 
 Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
 
->In the steps below, "ClientID" is the same as "Application ID" or "AppId".
-
-1. Open the `ToDoListClient\appsettings.json` file
-1. Find the app key `ClientId` and replace the existing value with the application ID (clientId) of the `WebApp-MultiTenant-ToDoListClient-v2` application copied from the Azure portal.
-1. Find the app key `TenantId` and replace the existing value with 'common'.
-1. Find the app key `Domain` and replace the existing value with your Azure AD tenant name.
-1. Find the app key `ClientSecret` and replace the existing value with the key you saved during the creation of the `WebApp-MultiTenant-ToDoListClient-v2` app, in the Azure portal.
-1. Find the app key `RedirectUri` and replace the existing value with the base address of the WebApp-MultiTenant-ToDoListClient-v2 project (by default `https://localhost:44321/`).
-1. Find the app key `TodoListScope` and replace the existing value with ScopeDefault.
-1. Find the app key `TodoListAppId` and replace the existing value with the application ID (clientId) of the `WebApi-MultiTenant-ToDoListService-v2` application copied from the Azure portal.
-1. Find the app key `TodoListBaseAddress` and replace the existing value with the base address of the WebApi-MultiTenant-ToDoListService-v2 project (by default `https://localhost:44351/`).
-1. Find the app key `AdminConsentRedirectApi` and replace the existing value with "https://localhost:44351/api/Home"
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `ToDoListClient\appsettings.json` file.
+1. Find the key `ClientId` and replace the existing value with the application ID (clientId) of `WebApp_MultiTenant_v2` app copied from the Azure portal.
+1. Find the key `TenantId` and replace the existing value with 'common'.
+1. Find the key `Domain` and replace the existing value with your Azure AD tenant domain, ex. `contoso.onmicrosoft.com`.
+1. Find the key `ClientSecret` and replace the existing value with the generated secret that you saved during the creation of `WebApp_MultiTenant_v2` copied from the Azure portal.
+1. Find the key `RedirectUri` and replace the existing value with the base address of `WebApp_MultiTenant_v2` (by default `https://localhost:44321/`).
+1. Find the key `TodoListScope` and replace the existing value with ScopeDefault.
+1. Find the key `TodoListAppId` and replace the existing value with the application ID (clientId) of `WebApi_MultiTenant_v2` app copied from the Azure portal.
```

Comment:
Reminder: update the `TodoListScope` `TodoListAppId` etc. to match appsettings.json